products/jbrowse-web/src/SessionLoader.ts

```diff
@@ -54,32 +53,25 @@ async function checkPlugins(pluginsToCheck: PluginDefinition[]) {
     if (isUMDPluginDefinition(p)) {
       return Boolean(
         storePlugins.plugins.find(
-          storePlugin =>
-            isUMDPluginDefinition(storePlugin) &&
-            (('url' in storePlugin &&
-              'url' in p &&
-              storePlugin.url === p.url) ||
-              ('umdUrl' in storePlugin &&
-                'umdUrl' in p &&
-                storePlugin.umdUrl === p.umdUrl)),
+          p =>
+            isUMDPluginDefinition(p) &&
```

Comment:
i didn't add checks for e.g. umdLoc here because since i didn't think our plugin store would use this